queries.sql

```diff
@@ -39,4 +39,15 @@ SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) FROM animals;
```

Comment:
- [OPTIONAL] 
You can consider adding the query questions and comments so that a visitor knows exactly what you are querying.

Reply:
@t-yanick
Okay. Thank you for the suggestions. I will implement it. 👍